src/datepicker/datepicker.js

```diff
@@ -649,7 +663,7 @@ function($compile, $parse, $document, $rootScope, $position, dateFilter, datePar
           if (isNaN(date)) {
             return undefined;
           } else {
-            return date;
+            return timelessJsonMode ? dateFilter(date, 'yyyy-MM-dd') : date;
```

Comment:
@wesleycho this parseDate function is added to the ngModel.$parsers for non-Html5DatrInputs; see line 699; I am setting the model, usually a date object, to a date string. I want the model to be a json compatable string so I can post it over ajax without any fuss -- and I also want to have it play nicely as a date in angular-land. The 'yyyy-MM-dd' format allows this to happen.